files/image_config/config-setup/config-setup

```diff
@@ -139,6 +139,9 @@ apply_tacacs()
         sonic-cfggen -j /etc/sonic/old_config/${TACACS_JSON_BACKUP} --write-to-db
         echo "Applied tacacs json to restore tacacs credentials"
         config save -y
+
+        # Change tacacs config file name, so tacacs config will not be revert when device reboot.
```

Comment:
>Change tacacs config file name, so tacacs config will not be revert when device reboot [](http://example.com/codeflow?start=10&length=86)

Should we just remove tacacs-config.service completely and simplified tacacs just as simple as part of the config_db.json?

Reply:
Complete remove the service is danger, because remove this service may cause SONiC missing TACACS config.
The OS upgrade scenario does not cover by sonic-mgmt test.